docs/Server.md

```diff
@@ -407,6 +418,17 @@ fastify.listen(3000, '127.0.0.1')
   })
 ```
 
+Specifying options without a callback is also supported:
+
+```js
+fastify.listen({ port: 3000, host: '127.0.0.1', backlog: 511 })
+  .then((address) => console.log(`server listening on ${address}`))
+  .catch(err => {
+    console.log('Error starting server:', err)
+    process.exit(1)
+  })
+```
+
```

Comment:
I would add an example with all the default options the user can set just before the [route](https://github.com/fastify/fastify/blob/f09e6c331297a924fd3422b417ee9a28708268a7/docs/Server.md#route) paragraph.

Ex:

```js
{
  port: 3000,
  host: '127.0.0.1',
  backlog: 511,
  exclusive: false,
  readableAll: false,
  writableAll: false,
  ipv6Only: false
}
```

Reply:
OK, will do it